### DISCUSSION TITLE
Markers disappear after saving them as code blocks

### DISCUSSION THREAD

**Discussion by kkstalls** [javalent/obsidian-leaflet#399] [Q&A]
I created a ton of markers and then saved them each as code blocks, but now none of them are showing up in the map. I made a custom map so I wonder if there is a layer issue, but I'm not seeing where to see or edit different layers.

After I had saved them all as code blocks, I went to the code of the map to be sure they were there. They're showing in the code, just not in the actual view. I tested it and it looks like the markers are disappearing when I create the marker, save it as a code block, and then go to the code itself.

I'm not seeing this in the other discussions or issues, but it might be related to something else or something really easy that I'm missing.

![Screenshot 2023-06-17 152036](https://github.com/javalent/obsidian-leaflet/assets/131550280/adf513d9-d9a9-41d1-953f-696092b64c3a)
![Screenshot (60)](https://github.com/javalent/obsidian-leaflet/assets/131550280/c8331d6d-dd34-4986-ba90-1303335dc1e1)

**Comment by KillbotGamma** [comment]
@kkstalls I'm seeing this issue as well. They do show, but only at like the center zoom level. My map is set to min-zoom -1.5, max-zoom 1.5, and they only show up on the 0 zoom level. can you test if you're seeing this as well?


Edit: Heeey, i actually just figured this out. You'll need to update your markers in the following way depending on how your map zoom levels are set up. 
this:
`marker: Plannet,19.2431640625,86.811225,Nihahiri,,,`
should look like this instead:
`marker: Plannet,19.2431640625,86.811225,Nihahiri,,-2,10`
replacing [min-zoom] and [max-zoom] with the corres

**Comment by MrYod** [comment]
Just remove the last 3 ","

So, eg: marker: Planet,19.2431640625,86.8125,Nihahiri
instead of: Planet,19.2431640625,86.8125,Nihahiri,,,


It's stupid, but that's what solves the bug.
